client/src/main/java/io/split/client/utils/FlagSetSanitizer.java

```diff
@@ -0,0 +1,46 @@
+package io.split.client.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public final class FlagSetSanitizer {
```

Comment:
please follow the spec and use the same name

Reply:
Updated